egs/tedlium/s5_r3/local/chain/run_tdnn_1c.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
+
+# 1g is as 1f but moving to a factorized TDNN (TDNN-F) model, re-tuning it, and
```

Comment:
this comment is no longer relevant.  you should explain that it's copied from the 1g script in the tedlium/s5_r2 setup, and that it replaces the current run_tdnn_1b.sh script.  (You can rename the old run_tdnnf.sh script as tuning/run_tdnn_1b.sh so we can keep track of it).